### ISSUE TITLE
Unable to import cognito user pool client

### ISSUE THREAD

**Issue by repl-chris** [terraform-providers/terraform-provider-aws#3753]
### Terraform Version
```
Terraform v0.11.3
+ provider.aws v1.11.0
```

### Affected Resource(s)
aws_cognito_user_pool_client

### Terraform Configuration Files
resource "aws_cognito_user_pool" "pool" {
&nbsp;&nbsp;&nbsp;&nbsp;name = "pool"
}
resource "aws_cognito_user_pool_client" "client" {
&nbsp;&nbsp;&nbsp;&nbsp;name = "client"
&nbsp;&nbsp;&nbsp;&nbsp;user_pool_id = "${aws_cognito_user_pool.pool.id}"
}

### Debug Output
https://gist.github.com/repl-chris/7215ee65e99e7ead2caf8100b0b1101d

### Expected Behavior
It should import successfully

### Actual Behavior
It appears to import successfully, but then errors out on 'Refreshing state...'
```
C:\temp> terraform import aws_cognito_user_pool_client.client 3v1ugef2oda5amrfsv9faegkog
aws_cognito_user_pool_client.client: Importing from ID "3v1ugef2oda5amrfsv9faegkog"...
aws_cognito_user_pool_client.client: Import complete!
  Imported aws_cognito_user_pool_client (ID: 3v1ugef2oda5amrfsv9faegkog)
aws_cognito_user_pool_client.client: Refreshing state... (ID: 3v1ugef2oda5amrfsv9faegkog)

Error: aws_cognito_user_pool_client.client (import id: 3v1ugef2oda5amrfsv9faegkog): 1 error(s) occurred:

* import aws_cognito_user_pool_client.client result: 3v1ugef2oda5amrfsv9faegkog: aws_cognito_user_pool_client.client: InvalidParameter: 1 validation error(s) found.
- minimum field size of 1, DescribeUserPoolClientInput.UserPoolId.
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Comment out the aws_cognito_user_pool_client resource from the config listed above
2. `terraform apply`
3. Add a client manually (via the AWS console or whatever)...take note of its ID
4. Uncomment out the aws_cognito_user_pool_client
5. `terraform import aws_cognito_user_pool_client.client <your client ID here>`

**Comment by loivis** [comment]
submitted pr to include `user-pool-id` for import

**Comment by arocki7** [comment]
I have got the same problem.

**Comment by bflad** [comment]
The fix for this has been merged in and will release with v1.16.0 of the AWS provider, likely tomorrow.

**Comment by bflad** [comment]
This has been released in [version 1.16.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1160-april-25-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
